Untitled-Engine/Src/Untitled/Core/Events/ApplicationEvent.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2020 [Untitled Games]
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE', which is part of this source code package.
+// 
+// [+ :]
+
+#pragma once
+
+#include "Event.h"
+
+namespace ue {
+	
+	class WindowClosedEvent : public Event {
+	public:
+		WindowClosedEvent() = default;
+		EVENT_CLASS_TYPE(WindowClosed)
+		EVENT_CLASS_CATEGORY(int(EventCategory::EventCategoryApplication))
+	};
+
+	class WindowResizeEvent : public Event {
+	public:
+		WindowResizeEvent(uint32_t width, uint32_t height) : m_Width(width), m_Height(height) {}
+		_NODISCARD inline uint32_t GetWidth() const { return m_Width; }
+		_NODISCARD inline uint32_t GetHeight() const { return m_Height; }
```

Comment:
Is there any point in declaring these as inline? They don't need inline for in-class definition and the compiler will inline them anyway.

Reply:
Removed unecessary inlines.